Apps/CocoaPods-FCM/src/View/DashboardView.swift

```diff
@@ -40,107 +37,120 @@ struct DashboardView: View {
                 }
             }
 
-            VStack(spacing: 10) {
+            VStack(spacing: 15) {
+                if let loggedInUserEmail = userManager.email {
+                    Text(loggedInUserEmail)
+                }
                 Text("What would you like to test?")
 
-                ColorButton(title: "Send Random Event") {
-                    CustomerIO.shared.track(
-                        name: String.random,
-                        data: [
-                            "randomAttribute": String.random,
-                            "random_attribute": String.random
-                        ]
+                ColorButton("Send Random Event") {
+                    switch Int.random(in: 0 ..< 3) {
+                    case 0:
+                        CustomerIO.shared.track(name: "Order Purchased")
+                    case 1:
+                        CustomerIO.shared.track(
+                            name: "movie_watched",
+                            data: [
+                                "movie_name": "The Incredibles"
+                            ]
+                        )
+                    default: // case 2
+                        CustomerIO.shared.track(
+                            name: "appointmentScheduled",
+                            data: [
+                                "appointmentTime": Calendar.current.date(byAdding: .day, value: 7, to: Date())!.epochNoMilliseconds
+                            ]
+                        )
+                    }
+
+                    showRandomEventSentAlert.toggle()
+                }
+                .setAppiumId("Random Event Button")
+                .alert(isPresented: $showRandomEventSentAlert) {
+                    Alert(
+                        title: Text("Random event sent"),
+                        dismissButton: .default(Text("OK"))
                     )
-                }.setAppiumId("Random Event Button")
-                ColorButton(title: "Send Custom Event") {
-                    showingCustomEventSheet.toggle()
-                }.setAppiumId("Custom Event Button")
-                    .sheet(isPresented: $showingCustomEventSheet, content: {
-                        VStack(spacing: 15) {
-                            TextField("Event name", text: $customEventName)
-                            TextField("Property name", text: $customEventPropertyName)
-                            TextField("Property value", text: $customEventPropertyValue)
-                            Button("Send event") {
-                                CustomerIO.shared.track(name: customEventName, data: [
-                                    customEventPropertyName: customEventPropertyValue
+                }
+
+                Group { // hack to get around Swift UI's max of 10 Views rule.
```

Comment:
Why not create a group for all the buttons? Yes, there is a limit of 10 views in the various Stacks, but isn't the Group a structure to collect similar views (elements)? I might be wrong here, but I think it isn't exactly a hack if used with a purpose.